please.sh

```diff
@@ -2851,12 +2852,15 @@ bundle_pdbs () { # [--directory=<artifacts-directory] [--unpack=<directory>] [--
 	architectures="i686 x86_64"
```

Comment:
I'd actually be in favor of only supporting a single architecture at a time in `bundle_pdbs` to make it consistent with most of the other stuff we've been refactoring. WDYT?